packages/x-components/src/x-modules/next-queries/store/actions/fetch-next-query-preview.action.ts

```diff
@@ -1,27 +1,32 @@
 import { NextQueriesXStoreModule } from '../types';
 import { XPlugin } from '../../../../plugins/x-plugin';
+import { createOrigin } from '../../../../utils/index';
```

Comment:
Avoid importing from barrel

```suggestion
import { createOrigin } from '../../../../utils/origin';
```